packages/dds/tree/src/test/simple-tree/node-kinds/object/objectNode.spec.ts

```diff
@@ -918,6 +927,61 @@ describeHydration(
 			const builderB: BuildB = { a: 1, thisDoesNotExist: 5 };
 		});
 
+		it("Custom Keys", () => {
```

Comment:
No changes to any of the production code here, I just noticed a lack of coverage when working on the import/export tests.